src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,191 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.pagefactory.AjaxElementLocatorFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+
+public class BasePage {
+
+//    private String isSuccessLocator =".success.show";
+//    private String playButtonAreaLocator = "[class='album-thumb']";
+//    private String playButtonLocator = "[data-testid='play-btn']";
+//    private String pauseButtonLocator = "[data-testid='pause-btn']";
+//    private String linkAllSongsLocator = "[class='songs']";
```

Comment:
I **recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.

Reply:
Hi, ok, i've deleted it